src/Server.cpp

```diff
@@ -221,21 +221,21 @@ void Server::readFromClient(Connection &conn, size_t &i, Parser &parser, HTTPReq
 	if (parser.getHeadersComplete() && request.getMethod() == "GET")
 		conn.setHasFinishedReading(true);
 
-	if (request.getMethod() == "GET" || request.getMethod() == "DELETE" || request.getMethod() == "SALAD")
+	if (request.getMethod() == "GET" || request.getMethod() == "SALAD")
 		Debug::log("GET request, no body to read", Debug::NORMAL);
 	else
-		handlePostRequest(conn, parser, request, response);
+		handlePostAndDelete(conn, parser, request, response);
 }
 
-void Server::handlePostRequest(Connection &conn, Parser &parser, HTTPRequest &request, HTTPResponse &response)
+void Server::handlePostAndDelete(Connection &conn, Parser &parser, HTTPRequest &request, HTTPResponse &response)
 {
 	if (parser.getIsChunked() && !conn.getHasReadSocket())
 	{
 		Debug::log("Chunked body", Debug::NORMAL);
-		if (!conn.readChunkedBody(parser))
+		if (!conn.readChunkedBody(parser) && (!conn.readBody(parser, request, response)))
```

Comment:
Why if the request is chucked, we read in a normal way?

Reply:
@dantol29  Sorry this is an error, I went through this function (add some point had a copy of it) to identify a problem and I didn't knew it well so I modified something I shouldn't have